### PR TITLE
Dev/180 New Problem Editor: Visible / New Problem Editor: Description. 

### PR DESCRIPTION
**Pull request by Wally5077**
resolve: #173 #180

### CODE REVIEW

File: src/app/problem/editor/right/Visible.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
 import {EditorButton} from "../../commons/EditorButton";
 import Block from "./Block";
 import {useState} from "react";
+import {useProblemEditorContext} from "../ProblemEditorContext";
+import {problemService} from "../../../../services/services";
 
 
 // TODO: This feature has low priority. Please see the related issue#180.
 const Visible = () => {
```
Comment (Chaoyuuu) [inline src/app/problem/editor/right/Visible.js:9]: Please remove this TODO.

File: src/services/ProblemService.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ export class ProblemService extends AbstractService {
         return this.axios.patch(`/api/problems/${problemId}`, {problemId, description});
     }
 
+    async modifyProblemVisible(problemId, visible) {
+        return this.axios.patch(`/api/problems/${problemId}`, {problemId, visible});
+    }
+
```
Comment (Chaoyuuu) [inline src/services/ProblemService.js:25]: Please rename the following methods:
`modifyProblemTitle` to `updateProblemTitle`
`modifyProblemDescription` to `updateProblemDescription`
`modifyProblemVisible` to `updateProblemVisible`

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review Done! Thanks~

**Review by Chaoyuuu — APPROVED** [review]
LGTM 👍🏻

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Review by nyngwang — APPROVED** [review]
LGTM.

**Pull request merged**